FlowerShop/FlowerShop/Scenes/OrderDetails/OrderDetailsService.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  OrderDetailsService.swift
+//  FlowerShop
+//
+//  Created by Andrei Oltean on 23/02/2020.
+//  Copyright © 2020 Andrei Oltean. All rights reserved.
+//
+
+import UIKit
+
+enum ImageResponse {
+    case success(UIImage)
```

Comment:
Is it ok to have UIKit components here? could we find another option?

Reply:
I have changed the code so that the order details service does not import UIKit. However, as the example was simple and I needed to pass a single UIImage to the view controller, I felt it was rather an overhead to create another view model just to encapsulate the transformation from Data to UIImage inside it (even though this is what would probably happen for more complex view models that would pass along more data to the view controller)